hubblestack/extmods/modules/conf_publisher.py

```diff
@@ -0,0 +1,46 @@
+
+import logging
+import hubblestack.splunklogging
+import copy
+import time
+
+log = logging.getLogger(__name__)
+
+def publish():
+    log.info('Started publishing config to splunk')
+    
+    initialize_splunklogging()
+    filtered_conf=filter_config()
```

Comment:
I think you're going to find that `__opts__` is huge, even after filtering out tokens/passwords. What would you think about having `publish()` take a list of opts to log? That way you can just collect the data you want and keep the noise level low?